Makefile

```diff
@@ -30,8 +30,17 @@ ifeq ($(ARCH),ppc64le)
 	LIB_DIR=powerpc64le-linux-gnu
 	CC=powerpc64le-linux-gnu-gcc
 endif
+ifeq ($(ARCH),s390x)
+	LIB_DIR=s390x-linux-gnu
+	CC=s390x-linux-gnu-gcc
+endif
 GOARM=6
-KUBE_CROSS_TAG=v1.6.2-2
+ifeq ($(ARCH),s390x)
+	# kube-cross:v1.7.4-1 supports s390x
+	KUBE_CROSS_TAG=v1.7.4-1
```

Comment:
You did :smile: I now need to make a decision about whether we want consistency across our platforms (at a cost of a bit of reduced stability) or whether to take this change as is but to have the extra complexity in the builds.

I don't think it needs to be decided right now, but should be before the next flannel release.